yt/visualization/plot_container.py

```diff
@@ -20,6 +21,36 @@
 
 from ._commons import validate_image_name
 
+try:
+    import cmocean  # noqa
+except ImportError:
+    cmocean = None
+
+CMOCEAN_DEPR_MSG = textwrap.dedent(
+    """
+    It looks like you requested a colormap from the cmocean library. In the future, yt won't "
+    recognize this bare (unprefixed) name. Please follow the recommended usage from cmocean's doc:
+    add an explicit `import cmocean` to your script, and prefix their colormap names with 'cmo.'
+    See our release notes for why this change was necessary."""
```

Comment:
```suggestion
    WARNING: It looks like you requested a colormap from the cmocean library. In the future, yt won't
    recognize this bare (unprefixed) name. Please follow the recommended usage from cmocean's doc:
    add an explicit `import cmocean` to your script, and prefix their colormap names with 'cmo.'
    See our release notes for why this change was necessary."""
```

Reply:
well it's only ever displayed as a `VisibleDeprecationWarning`, I don't think it's necessary to add "WARNING" to the text itself. @chummels

Reply:
OK, no worries.